content/rest/usage.textile

```diff
@@ -188,7 +188,7 @@ blang[flutter].
 
 "View our client library SDKs feature support matrix":https://ably.com/download/sdk-feature-support-matrix to see the list of all the available features.
 
-The supported "client options are described below":#client-options.
+The supported "client options are described below":/api/rest-sdk/types#client-options.
```

Comment:
I would change the link text. It is true that the client options used to be below, but we split them off so that is no longer true. Perhaps something like: 

The supported client options are described in the API reference documentation.

Reply:
Sorry @tbedford I don't know how I missed that. I've changed it now. Also fixed in the other versions.